### PR TITLE
If remove responsive core

### PR DESCRIPTION
**Pull request by Ianfeather**
Removes the .responsive flag from (almost) everything under `/stylesheets/core`.

This basically means secondary nav and footer need checking

### CODE REVIEW

File: app/assets/stylesheets/core/_grid.sass [hunk 1]
```diff
@@ -316,26 +316,22 @@
   box-sizing: border-box
   -moz-box-sizing: border-box
```
Comment (i-like-robots) [inline app/assets/stylesheets/core/_grid.sass]: Autoprefixer should handle all of our prefixing needs

File: app/assets/stylesheets/core/layouts/modern/_footer.sass [hunk 2]
```diff
@@ -11,77 +11,55 @@
...
-  margin: 0 20px 15px 0
-  .responsive &
-    margin: 0
-    +respond-to(wide-view)
-      float: left
-      margin: 0 20px 15px 0
+  border-bottom: 1px solid $input-border
+  +respond-to(wide-view)
+    border: none
+    padding: 0
```
Comment (i-like-robots) [inline app/assets/stylesheets/core/layouts/modern/_footer.sass]: Not sure there's any padding to override

File: app/assets/stylesheets/core/layouts/modern/_footer.sass [hunk 3]
```diff
@@ -11,77 +11,55 @@
...
+  +respond-to(wide-view)
+    float: right
+    width: 240px
+    padding: 0
+    margin-right: 10px
+
+  @if $is-ie == true
+    width: 200px
 
   .btn--green
```
Comment (i-like-robots) [inline app/assets/stylesheets/core/layouts/modern/_footer.sass]: we should really author buttons/badges etc with both the object and modifier (`class="btn btn--green"`) so we're not targeting specific visual stuff.

File: app/assets/stylesheets/core/layouts/modern/_footer.sass [hunk 4]
```diff
@@ -93,36 +71,28 @@
...
-    display: none
-    .responsive &
-      display: block
-      +respond-to(wide-view)
-        display: none
+    +respond-to(wide-view)
+      display: none
 
   .accordion__title--footer
     float: right
```
Comment (i-like-robots) [inline app/assets/stylesheets/core/layouts/modern/_footer.sass]: I think the accordion is only used for the site footer, it's begging for a clean out soon.

File: app/assets/stylesheets/core/layouts/modern/_footer.sass (hunk 3 above)
Comment (Ianfeather) [inline app/assets/stylesheets/core/layouts/modern/_footer.sass]: It has the object and modifier. I've added a component specific modifier now and targeted that.

File: app/assets/stylesheets/core/layouts/modern/_footer.sass (hunk 4 above)
Comment (Ianfeather) [inline app/assets/stylesheets/core/layouts/modern/_footer.sass]: Totally

Comment (Ianfeather) [inline app/assets/stylesheets/core/layouts/modern/_footer.sass]: Could be much much simpler

**Comment by i-like-robots** [comment]
:+1:

**Pull request merged**